external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -145,7 +148,8 @@ public static void main(String args[]) {
                 NewKafkaSpoutOffsetQuery newKafkaSpoutOffsetQuery =
                     new NewKafkaSpoutOffsetQuery(commandLine.getOptionValue(OPTION_TOPIC_LONG),
                                                  commandLine.getOptionValue(OPTION_BOOTSTRAP_BROKERS_LONG),
-                                                 commandLine.getOptionValue(OPTION_GROUP_ID_LONG), securityProtocol);
+                                                 commandLine.getOptionValue(OPTION_GROUP_ID_LONG), securityProtocol,
```

Comment:
I suggest adding a check in the old spout if block and throw an error if this option provided with old spout.